recipes/xorg-proto/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "pkg_config"
```

Comment:
.pc files are not used afterwards, but I guess you want to locally check their content?

Reply:
I added it for myself to visually check whether the pc files looked correct.
But I didn't program a test in cmake.

Reply:
didn't see https://github.com/conan-io/conan-center-index/pull/6473#discussion_r676056491